### ISSUE TITLE
 Could not read ETS export file

### ISSUE THREAD

**Issue by nanosonde** [pakerfeldt/knx-mqtt-bridge#2]
I am getting this.

The group address were exported with ETS 5.6.5.
I have clicked on the root node within ETS and then selected "GroupAddress export".
As output format I have just selected "XML" and not "XML (ETS4 format).

Do you support ETS5 XML files yet?

**Comment by pakerfeldt** [comment]
It supports ETS5. I use it myself. Would you mind sharing the XML file? You can mail it to patrik.akerfeldt@gmail.com. Otherwise, maybe there are some unrecognized characters in names I haven't accounted for? Try removing chunks from the XML file until it's parsable and maybe you can figure out which part it doesn't understand so I can fix it.

**Comment by nanosonde** [comment]
Hi!
Thanks for the quick reply.

```
<FEFF><?xml version="1.0" encoding="utf-8" standalone="yes"?>
<GroupAddress-Export xmlns="http://knx.org/xml/ga-export/01">
  <!--GroupRange:-->
  <!--  @Name: The name of the group range (1 to 255 characters)-->
  <!--  @RangeStart: The inclusive start address of the range as specified in free group address style-->
  <!--  @RangeEnd: The inclusive end address of the range as specified in free group address style-->
  <!--  @Description: The group ranges' description if available-->
  <!--GroupAddress:-->
  <!--  @Name: The name of the group range (1 to 255 characters)-->
  <!--  @Address: The address as specified either in three-level (1/2/3 or two-level (1/3) or numeric (12345) or hexadecimal ($1Fa3 or 0x1fa3) group address value-->
  <!--  @Central: True if the group address has the Central flag set-->
  <!--  @Unfiltered: True if the group address has the Unfiltered flag set-->
  <!--  @Description: The group addresses' description if available-->
  <!--  @DTPs: The Data Point Types associated with the group address if available, comma separated, e.g. '1.001, 1.002'-->
  <GroupRange Name="Licht" RangeStart="2048" RangeEnd="4095">
    <GroupRange Name="Umwelt/Zentral" RangeStart="2048" RangeEnd="2303">
      <GroupAddress Name="Licht gesamt schalten" Address="1/0/0" Description="Testbeschreibung" DPTs="DPST-1-1" />
    </GroupRange>
  </GroupRange>
</GroupAddress-Export>
```

I have stripped it down to a single GA which is also not working.

**Comment by pakerfeldt** [comment]
I'm pretty sure it's the comments. Try removing those and see if it helps. I will make sure to fix that issue if that's the case.

**Comment by pakerfeldt** [comment]
Thanks for the report. This should be fixed in 0.0.18 which I just published to npm.

**Comment by nanosonde** [comment]
Hi! 
It is loading the ETS XML export file without any errors.
Thanks for providing the quick fix.
